04/app.js

```diff
@@ -5,4 +5,54 @@ const menuItems = [
     {text: 'start', url: '/'},
     {text: 'galeria', url: '/gallery'},
     {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+];
+
+const navElement = document.querySelector("nav");
+
+if(navElement){
+    const ulElement = document.createElement("ul");
+    navElement.appendChild(ulElement);
+    
+    for(let i = 0; i < 3; i++) {
```

Comment:
Tutaj mało elastycznie. Jeśli dodasz jakiś element do menu to pętla też będzie musiała zostać zmienione.
Lepiej napisać `i < menuItems.length`